src/EventStore.Core/LogV3/DummyPartitionManager.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace EventStore.Core.LogV3
+{
+	public class DummyPartitionManager : IPartitionManager {
```

Comment:
just for the pattern, this could be called `LogV2PartitionManager` and be moved to the `Core.LogV2` namespace